src/main/java/jenkins/MasterToSlaveFileCallable.java

```diff
@@ -7,6 +7,7 @@
 
 /**
  * {@link FileCallable}s that are meant to be only used on the master.
+ * @deprecated Use a core dependency of 1.580.1 or later rather than this library.
  */
 public abstract class MasterToSlaveFileCallable<T> implements FileCallable<T>, RoleSensitive {
```

Comment:
The annotation is implied by the Javadoc tag in the sense that (AFAIK) all tools which interpret the annotation also honor the Javadoc tag. IIRC there is a lint option for `javac` to make the two align, i.e., that every member either has both or neither. If you care about the existence of the annotation, I would suggest making the parent POM (or this one?) enforce that lint. That said, if this is `:bug:` I will fix it.